main.go

```diff
@@ -659,6 +660,7 @@ func getTempEtcdClient(ctx context.Context, endpoints []string, etcdCfg *embetcd
 		if err == nil {
 			break
 		}
+		logger.Log(log.Err, err, "unable to create temp cli")
```

Comment:
Maybe be a bit more explicit in that error message; users won't understand what this means.